src/dsp_tools/commands/excel2json/models/list_node.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
```

Comment:
I ended up not going for pydantic, as there is no way to validate it without raising an error.
When I create the lists, I don't want to catch errors but match the class type that was returned. And if it is of type "...Problem" then I will append it to a problem list, which I will check in the end. Copilot told me that a class method the easiest way to create this functionality.